public/cli/lib/rendering/renderFrames.ts

```diff
@@ -0,0 +1,42 @@
+import { record } from './recordFrames';
+import fs from 'fs';
+import path from 'path';
+import { AnimationSettings, Timings, NotifyEvent, ProjectData } from '../../../models';
+import { allowedNodeEnvironmentFlags } from 'process';
```

Comment:
Looks like this line is unused, remove it.

Reply:
commented out because I need to check later how we should use this, if at all.